pkgs/by-name/nc/nccm/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, python3, fetchFromGitHub }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "nccm";
+  version = "1.3.0";
+  format = "other";
+  src = fetchFromGitHub {
+    owner = "flyingrhinonz";
+    repo = "nccm";
+    rev = version;
+    hash = "sha256-1leAyYrkilPjHt3R6AKtGXBDmXfERpH8Y4BhzToGWEs=";
+  };
+  propagatedBuildInputs = with python3.pkgs; [ pyyaml ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp nccm/nccm $out/bin
+
+    mkdir -p $out/share/nccm
+    cp nccm/nccm.yml $out/share/nccm
```

Comment:
But the share directory should not be used like this. This issue should probably be addressed upstream.